src/field/field_factory.cxx

```diff
@@ -222,10 +222,11 @@ Field3D FieldFactory::create3D(FieldGeneratorPtr gen, Mesh* localmesh, CELL_LOC
   }
   };
 
-  if (transform_from_field_aligned and localmesh->canToFromFieldAligned()) {
+  if (transform_from_field_aligned
+        and result.getCoordinates()->getParallelTransform().canToFromFieldAligned()) {
```

Comment:
Looks like this could do with being a free function as well

Reply:
Free function would be nicer, but this is the only place it's called in the whole library - is it worth having? In practice `canToFromFieldAligned` is checking whether we're using FCI or not. I guess a user code would either use FCI or not, so wouldn't be likely to need a check like this?